### ISSUE TITLE
[TODO]  Add shuffle test

### ISSUE THREAD

**Issue by VitalyFedyunin** [pytorch/data#589]
This issue is generated from the TODO line
https://github.com/pytorch/data/blob/2f29adba451e1b87f1c0c654557d9dd98673fdd8/./test/test_dataloader2.py#L164

**Comment by DongyuXu77** [comment]
I could add the shuffle test when I  fix the #753

**Comment by DongyuXu77** [comment]
Is anyone take it? Maybe I could work on it

**Comment by ejguan** [comment]
@DongyuXu77 Thank you for asking. This is actually blocked by determinism support for DataLoader2, which I am currently working on. 
I will assign the Issue to myself.

**Comment by DongyuXu77** [comment]
Oh, thanks to your reply

**Comment by ejguan** [comment]
Tests are completed. But, I still need to remove the `TODO` comment in the test
